addons/hr_expense/static/src/scss/hr_expense.scss

```diff
@@ -18,3 +18,19 @@
         }
     }
 }
+
+.o_dropzone {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    background-color: #AAAA;
+    z-index: 2;
+    left: 0;
+    top: 0;
+    i {
+      justify-content: center;
+      display: flex;
+      align-items: center;
+      height: 100%;
+    }
+}
```

Comment:
New line missing at the end of the file. These can be avoided by enabling the following setting in VSCode
![image](https://user-images.githubusercontent.com/88150098/183927092-a3bc65b2-433b-4372-b2e1-05e4b46629bc.png)